homeassistant/components/sensor/ecobee.py

```diff
@@ -0,0 +1,101 @@
+"""
+homeassistant.components.sensor.ecobee
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This sensor component requires that the Ecobee Thermostat
+component be setup first. This component shows remote
+ecobee sensor data.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.ecobee/
+"""
+from homeassistant.helpers.entity import Entity
+import json
+import logging
+import os
+
+SENSOR_TYPES = {
+    'temperature': ['Temperature', '°F'],
+    'humidity': ['Humidity', '%'],
+    'occupancy': ['Occupancy', '']
+}
+
+_LOGGER = logging.getLogger(__name__)
+
+ECOBEE_CONFIG_FILE = 'ecobee.conf'
+
+
+def config_from_file(filename, config=None):
```

Comment:
There is no writing done in this file. Please simplify this method to only the used parts.